pkg/app/piped/cloudprovider/cloudrun/cloudrun.go

```diff
@@ -146,6 +148,30 @@ func (s *Service) ActiveRevisionNames() []string {
 	return ret
 }
 
+func (s *Service) HealthStatus() (model.CloudRunResourceState_HealthStatus, string) {
+	var (
+		status     = s.Status
+		errMessage = "Unexpected error while calculating: %s"
+	)
+	if status == nil {
+		msg := fmt.Sprintf(errMessage, "unable to find status")
+		return model.CloudRunResourceState_UNKNOWN, msg
```

Comment:
```suggestion
		return model.CloudRunResourceState_UNKNOWN, fmt.Sprintf(errMessage, "unable to find status")
```
nits, same with the below

Reply:
Thanks for reviewing.
I fixed from reviews. PTAL when you can do.